types/patches/0032-upstream-expo-modules-core-Comment-suppress-an-unres.patch

```diff
@@ -0,0 +1,24 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Chris Bobbe <cbobbe@zulip.com>
+Date: Mon, 25 Jul 2022 14:22:06 -0700
+Subject: [upstream] expo-modules-core: Comment/suppress an unresolved r-n-web
+ import
+
+---
+ .../build/SyntheticPlatformEmitter.web.js.flow                  | 2 ++
+ 1 file changed, 2 insertions(+)
+
+diff --git types/expo-modules-core/build/SyntheticPlatformEmitter.web.js.flow types/expo-modules-core/build/SyntheticPlatformEmitter.web.js.flow
+index 9c662b1c1..02859add9 100644
+--- types/expo-modules-core/build/SyntheticPlatformEmitter.web.js.flow
++++ types/expo-modules-core/build/SyntheticPlatformEmitter.web.js.flow
+@@ -1,4 +1,6 @@
+ /* @flow
+  * @generated by TsFlower
+  */
++/* $FlowFixMe[cannot-resolve-module]: expo-modules-core should (peer-)depend
++   on react-native-web if it wants this to be a meaningful type. */
```

Comment:
Hmm, or perhaps better: we could delete this file entirely, say in `tools/tsflower` after running TsFlower (in `run_only`.)

In general we don't ever care about a `.web.js.flow` file, because it isn't applicable when not on web. In a future where TsFlower has a feature to tell it to skip some files, as suggested here:
```
    # TODO: Send r-n-tab-view a .npmignore patch to stop shipping this directory.
    #   Also perhaps give TsFlower a way to tell it things to skip.
    rm -rf "${rootdir}"/types/"${package}"/lib/typescript/example/
```
we'd have entries there like `*.web.d.ts`.

Reply:
Good idea! #5478